### PR TITLE
run trie generation thread on background QOS

### PR DESCRIPTION
**Pull request by tomerd**
motivation: build the trie should not interrupt other threads

changes: set QOS of trie thread to background

### CODE REVIEW

**Comment by tomerd** [comment]
@swift-ci please smoke test

**Comment by tomerd** [comment]
cc @yim-lee

File: test [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.
```
Comment (yim-lee) [inline test:1]: Is this file supposed to be checked in?

**Review by yim-lee — APPROVED** [review]

File: test (hunk 1 above)
Comment (tomerd) [reply test]: removed

**Comment by tomerd** [comment]
@swift-ci please smoke test

**Pull request merged**